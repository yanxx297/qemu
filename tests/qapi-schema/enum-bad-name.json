# FIXME: we should ensure all enum names can map to C
{ 'enum': 'MyEnum', 'data': [ 'not^possible' ] }
