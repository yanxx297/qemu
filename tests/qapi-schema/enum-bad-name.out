[OrderedDict([('enum', 'MyEnum'), ('data', ['not^possible'])])]
[{'enum_name': 'MyEnum', 'enum_values': ['not^possible']}]
[]
